### ISSUE TITLE
[1][HarukaMa] Added smartcoin feed producers removed from list on tab switch

### ISSUE THREAD

**Issue by grctest** [bitshares/bitshares-ui#1024]
Hey,

When you add accounts to the smartcoin feed producer list, but navigate to another tab before clicking 'update asset', the new accounts will be removed from the list.

Just happened when I added 20 accounts, checked the other tabs then hit 'update asset' and realized that it hadn't updated the feed producer list.

Best regards,
CM.

**Comment by wmbutler** [comment]
Would you mind posting a licecap?

**Comment by grctest** [comment]
<img src="https://i.imgur.com/JWJAN2W.gif"/>

**Comment by HarukaMa** [comment]
I'd like to claim this.

**Comment by HarukaMa** [comment]
I've decided to combine the feed producer settings into the whole asset update page instead of letting it having separate update and reset button. By doing this I've also added page-wide change detector which automatically enable / disable the "update asset" button, and made it only updates asset basic information if needed instead of always updating even if no changes are made. (It has cost me a few BTS fees when playing with my test asset before.)

Still finishing the table update part, will PR soon.
